libtransmission/announcer.c

```diff
@@ -1277,7 +1279,9 @@ static void on_announce_done(tr_announce_response const* response, void* vdata)
             {
                 /* the queue is empty, so enqueue a perodic update */
                 i = tier->announceIntervalSec;
```

Comment:
remove this line since it doesn't do anything any more

Reply:
Done.